ClassificationNeuralNetwork.java

```diff
@@ -62,4 +82,41 @@ public static void readDataFromFile()
 					+ currLineNum);
 		}
 	}
+	
+	public static void putDataInArrayList()
+	{
+		for(int i=0; i<numLines; i++)
+		{
+			double[] toAdd= new double[10];
+			
+			toAdd[0]=radii[i];
+			toAdd[1]=textures[i];
+			toAdd[2]=perimeters[i];
+			toAdd[3]=area[i];
+			toAdd[4]=smoothness[i];
+			toAdd[5]=compactness[i];
+			toAdd[6]=concavity[i];
+			toAdd[7]=concavePnts[i];
+			toAdd[8]=symmetry[i];
+			toAdd[9]=fractalDim[i];
+			
+			listOfTestData.add(toAdd);
+			listOfTestClassifications.add(classification[i]);
+		}
+	}
+	
+	public static void shuffle()
+	{
+		int eightyPercentOfData= (int) (listOfTestData.size()*.8);
+		
+		for(int i=0; i<eightyPercentOfData; i++)
+		{
+			int toRemove= (int) (Math.random()*listOfTestData.size());
+			listOfTrainingData.add(listOfTestData.remove(toRemove));
+			listOfTrainingClassifications.add(listOfTestClassifications.remove(toRemove));
+		}
+		
+		System.out.println("For reference, the size of the testing data is " + listOfTestData.size() + ", and the testing classification size is " + listOfTestClassifications.size() + ", which should be the same.");
```

Comment:
I would shorten this if possible. I think it's a useful line, but I think it might be more than we need. Such as "Testing Data Size: --         Testing Classification Size:               These Should be equal."

Reply:
Agreed. Making changes now.